AsyncAwait.Task2.CodeReviewChallenge/Services/PrivacyDataService.cs

```diff
@@ -1,12 +1,11 @@
-﻿using System.Threading.Tasks;
+﻿using AsyncAwait.Task2.CodeReviewChallenge.Constants;
 
 namespace AsyncAwait.Task2.CodeReviewChallenge.Services;
 
 public class PrivacyDataService : IPrivacyDataService
 {
-    public Task<string> GetPrivacyDataAsync()
+    public string GetPrivacyData()
     {
-        return new ValueTask<string>("This Policy describes how async/await processes your personal data," +
-                                     "but it may not address all possible data processing scenarios.").AsTask();
+        return PrivacyDataConstants.PersonalDataPolicy;
```

Comment:
PrivacyDataService.GetPrivacyDataAsync basically returns just a string. No need for Task. Also the string is extracted as a readonly constant in another class. Also ValueTask has it`s  performance and memory benefits over Task objects but should be used only in certain situations in which most of the return path of a method don`t have await statements. In our case it is not needed.